code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -43,6 +43,18 @@
 	var/obj/item/reagent_containers/food/snacks/deepfryholder/type = new(get_turf(src))
 	return type
 
+/obj/machinery/cooker/deepfryer/examine(mob/user)
+	. = ..()
+	if(emagged)
+		. += "<span class='warning'>The heating element is smoking slightly.</span>"
+
+/obj/machinery/cooker/deepfryer/emag_act()
+	if(!emagged)
+		to_chat(usr, "<span class='warning'>You short out the fryer's safeties, allowing non-food objects to be placed in the oil.</span>")
+		playsound(src, "sparks", 75, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
+		emagged = TRUE
+		return
```

Comment:
Is there anything stopping you from deep-frying your emag by accident after you emag it once, or is that kind of a feature? It looks like you shouldn't accidentally cook it the first time at least, which is good

Reply:
>Is there anything stopping you from deep-frying your emag by accident after you emag it once

Nope, you can deep fry your emag by clicking on the fryer again. I added the examine text `The heating element is smoking slightly.` to emagged fryers so traitors can check if a fryer's already been emagged before attempting to do so, while still leaving it possible if you _really_ want to get rid of your equipment through crispification.